src/locales/en.json

```diff
@@ -117,31 +117,31 @@
       "budget_header": "How can I manage my risk using a risk budget?",
-      "budget_explanation": "We decided an annual risk budget of a 1% chance of catching COVID per year was reasonable for most healthy people who are not in contact with vulnerable groups. To learn how we arrived at that number, <1>click here</1>. If you want to follow this 1%-risk-per-year budget, you’ll want your weekly budget of activities to sum to less than about 200 microCOVIDs each week. We’ve created a handy <3>spreadsheet</3> that lets you keep track of your activities over time."
+      "budget_explanation": "An annual risk budget of a 1% chance of catching COVID-19 per year is <1>our assumption</1> for what most healthy people not in contact with vulnerable groups consider reasonable. To follow this 1%-risk-per-year budget, your weekly budget should be less than 200 mCs each week. The handy <3>spreadsheet</3> lets you keep track of your activities over time."
     },
     "risk_tolerance_1_percent_label": "Standard Caution Budget",
-    "risk_tolerance_1_percent_explanation": "Budget: 1% chance of COVID per year (200 microCOVIDs per week) (suggested for healthy people NOT in close contact with more vulnerable people)",
+    "risk_tolerance_1_percent_explanation": "Budget: 1% chance of COVID per year (200 mCs per week) (suggested for healthy people NOT in close contact with more vulnerable people)",
     "risk_tolerance_point1_percent_label": "High Caution Budget",
-    "risk_tolerance_point1_percent_explanation": "Budget: 0.1% chance of COVID per year (20 microCOVIDs per week) (suggested if you or your close contacts are more vulnerable to COVID)",
+    "risk_tolerance_point1_percent_explanation": "Budget: 0.1% chance of COVID per year (20 mCs per week) (suggested if you or your close contacts are more vulnerable to COVID-19)",
     "intro": {
-      "whats_this": "<p>We’ve constructed a calculator that lets you estimate the risk of getting COVID from a wide range of activities, using the <1>best research available</1>. We hope this tool will help hone your intuition, lower your stress levels, and figure out good harm-reduction strategies.</p><p>The first step in protecting your family and community is to avoid getting COVID yourself.</p>",
+            "whats_this": "<p>The calculator sums up the risk of getting COVID-19 from a wide range of activities, using the <1>best research available</1>. This in the hopes of helping you hone your intuition, figuring out good harm-reduction strategies, and lowering your stress levels.</p><p>The first step in protecting your family and community is to avoid getting COVID yourself.</p>",
```

Comment:
This is wrong and something that users are already confused about - it doesn't "sum up" the risk.

Reply:
@beshaya To remove said doubt, what does it do?